src/vuex/modules/index/store.ts

```diff
@@ -1,230 +1,172 @@
 import * as getters from './getters'
-import {
-  dedupeRealtimeCollections,
-  splitRealtimeStoredCollections,
-  getRealtimeCollectionFromStorage
-} from '../../../services/data'
-import { removeIndex } from '../../../services/localStore'
-import Promise from 'bluebird'
+import { getIndexPosition } from '@/services/indexHelpers'
 import Vue from 'vue'
-import { IndexState, Index } from './types'
+import {
+  IndexState,
+  Index,
+  Collection,
+  CollectionType,
+  IndexCollectionPayload,
+  IndexCollectionsPayload,
+  IndexLoadingCollectionsPayload,
+  CreateCollectionPayload
+} from './types'
 import { createMutations, createModule, createActions } from 'direct-vuex'
 import { moduleActionContext } from '@/vuex/store'
 
 const state: IndexState = {
-  indexesAndCollections: {},
-  loadingIndexes: false,
-  loadingCollections: false
+  indexes: [],
```

Comment:
```suggestion
  indexes: Index[],
```

Reply:
Sorry, I don't remember why we didn't apply this...

Reply:
Because the `IndexState` interface already defined the type : https://github.com/kuzzleio/kuzzle-admin-console/blob/e2ff7711582abc27a7f3d0160f2ae5aacc19f96c/src/vuex/modules/index/types.ts#L84